File: eval/fixtures/cross-repo/acme-app/src/report.ts
import { overdue as isOverdue, type Task } from '@acme/core';

/** Locally aliased import: the call site says `isOverdue`, the package says `overdue`. */
export function overdueTitles(tasks: Task[], now: number): string[] {
  return tasks.filter((t) => isOverdue(t, now)).map((t) => t.title);
}

File: eval/fixtures/cross-repo-py/py-app/app/report.py
"""Reporting helpers over the taskcore API (locally aliased import)."""
from taskcore import overdue as is_overdue, Task


def overdue_titles(tasks: list[Task], now: int) -> list[str]:
    """The call site says `is_overdue`, the package exports it as `overdue`."""
    return [t.title for t in tasks if is_overdue(t, now)]
